Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -283,6 +299,7 @@ def keg_contain_absolute_symlink_starting_with?(string, keg)
         !absolute_symlinks_start_with_string.empty?
       end
 
+      sig { params(cellar: T.nilable(T.any(String, Symbol))).returns(T.nilable(T::Boolean)) }
```

Comment:
```suggestion
      sig { params(cellar: T.nilable(T.any(String, Symbol))).returns(T::Boolean) }
```

This may fail but: any method ending with `?` should always return a `T::Boolean` and never a nilable one.

Reply:
That works! I don't know how this got to nilable, because I agree with you.